addons/auth_oauth/res_config.xml

```diff
@@ -15,15 +15,24 @@
                             </div>
                             <div attrs="{'invisible':[('auth_oauth_google_enabled','=',False)]}">
                                 <blockquote>
-                                    To setup the signin process with Google, first you have to perform the following steps:<br/>
-                                    <br/>
-                                    - Go to the <a href="https://code.google.com/apis/console/">Google APIs console</a><br/>
-                                    - Ceate a new project<br/>
-                                    - Go to Api Access<br/>
-                                    - Create an oauth client_id<br/>
-                                    - Edit settings and set both Authorized Redirect URIs and Authorized JavaScript Origins to your hostname.<br/>
-                                    <br/>
-                                    Now copy paste the client_id here: <field name="auth_oauth_google_client_id" class="oe_inline" placeholder="e.g. 1234-xyz.apps.googleusercontent.com"/>
+                                    <p style="font-size:13px;color: rgb(76, 76, 76);">To setup the signin process with Google, first you have to perform the following steps:<br/>
```

Comment:
Please do not use inline / custom css .

why use <br/> ? can possible use <div> tag ?